Sources/Sentry/SentryFramesTracker.m

```diff
@@ -101,6 +122,17 @@ - (void)displayLinkCallback
             = 1 / (self.displayLinkWrapper.targetTimestamp - self.displayLinkWrapper.timestamp);
     }
 
+#    if SENTRY_TARGET_PROFILING_SUPPORTED
+    if (self.refreshRateTimestamps.count == 0
```

Comment:
`m`: Shouldn't we also guard this by`self.currentTracer.isProfiling`? We keep track of the refresh rate no matter if profiling is running or not.